### ISSUE TITLE
Homebrew cask for fully featured rclone

### ISSUE THREAD

**Issue by brettferdosi** [rclone/rclone#5373]
The current Homebrew formula for `rclone` builds a version of the binary that doesn't support mounting, even when `macfuse` is installed independently (https://github.com/rclone/rclone/issues/4775). I've created a Homebrew cask that downloads and installs a fully featured version of `rclone` from the project's GitHub releases page (and includes the `macfuse` cask as a dependency). I'm making an issue here so other Homebrew users who want to have a scriptable, fully-featured rclone install can know about it.

The cask is here: https://github.com/brettferdosi/homebrew-tap/blob/main/Casks/rclone-cask.rb - feel free to use it directly (though no guarantees about timely updates) or copy it into your own tap. I asked about getting this included in the main Homebrew cask tap but the response was negative: https://github.com/Homebrew/homebrew-cask/issues/106519.

**Comment by ivandeex** [comment]
@darthShadow Please take a look :pray:

**Comment by darthShadow** [comment]
The ideal solution would be to just move rclone from a formula to a cask which can, in turn, depend on macfuse. That way there are no duplicates and third-party repositories to deal with.

I don't think providing natively-compiled bottles provides any benefits over simply installing the release from Github, since rclone doesn't have much in the way of system dependencies, which should make the transition easier.

**Comment by brettferdosi** [comment]
I agree - maybe you'll get more traction asking for this change as rclone developers. Feel free to use my cask as a proposed starting point.

**Comment by Cnly** [comment]
Given the situation, it looks like rclone won't be getting into homebrew-cask anytime soon. Should rclone maintain a tap itself? We can use @brettferdosi's cask as a start and perhaps add a beta version as well.

**Comment by darthShadow** [comment]
I don't much care for fracturing the install process even further with our own maintained cask. If they have reached the point of reaching our website for the official cask then they may as well just follow the official install procedure and any future updates can be done manually via the auto-update command.

**Comment by darthShadow** [comment]
@BingoKingo From what I know, that won't work because homebrew doesn't support managing custom services, only default ones and there can never be a default mount.

Anyway, I think this topic can be closed. The homebrew team is not in favour of moving this formula to a cask and I am not in favour of maintaining our own cask when there are already multiple options available for getting the mount subcommand:

* Official install via https://rclone.org/install/
* Semi-official tap for the fuse-enabled formula: https://github.com/gromgit/homebrew-fuse
